packages/stryker/src/config/ConfigReader.ts

```diff
@@ -1,9 +1,10 @@
 import { Config } from 'stryker-api/config';
 import { StrykerOptions } from 'stryker-api/core';
 import * as fs from 'mz/fs';
-import * as log4js from 'log4js';
+import * as log4js from 'stryker-api/logging';
```

Comment:
we call it log4js here and import the entire module

Reply:
👍